build_scripts/create-mass-storage-image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BS="512"
+
+if (( $# != 2 )); then
+    >&2 echo -e "Usage:\n$0 mass_storage_contents_dir/ mass_storage.img"
+    exit 1
+else
+    CONTENTS_DIR=$1
+    IMAGE_FNAME=$2
+    echo "Creating $IMAGE_FNAME from $CONTENTS_DIR"
+fi
+
+set -e
+
+size=$(du --block-size=$BS -s "$CONTENTS_DIR" | awk '{print $1}')
```

Comment:
```suggestion
size=$(du -B $BS -s "$CONTENTS_DIR" | cut -f1)
```
BSD `du` (на macOS) не умеет в длинную опцию `--block-size`.

Reply:
готово